rpmio/rpmpgp.c

```diff
@@ -1104,6 +1104,8 @@ static int pgpVerifySelf(pgpDigParams key, pgpDigParams selfsig,
     return rc;
 }
 
+static const size_t RPM_MAX_OPENPGP_BYTES 65535 /* max number of bytes in a key */
```

Comment:
I think you're missing an equals sign here for assignment?

Reply:
That’s what I get for not even compile-testing my code.